### ISSUE TITLE
Cmd+1/2 doesn't focus on the input box of Interactive Window

### ISSUE THREAD

**Issue by corneliusroemer** [microsoft/vscode-jupyter#6602]
Issue #1293 is back. This is just a pointer to the upstream issue.  I'm opening this here just to point at the issue in the VS code tracker, because I was about to open it here and only after a lengthy search found out that the issue is already open over there.

Upstream: https://github.com/microsoft/vscode/issues/124788

**Comment by joyceerhl** [comment]
@corneliusroemer We are working on migrating the interactive window to use built-in VS Code APIs. It should address the ctrl+1/2 problem among others, and is currently in preview if you'd like to try it out. Details and instructions for trying it out here: https://code.visualstudio.com/updates/v1_58#_jupyter-interactive-window

**Comment by corneliusroemer** [comment]
@joyceerhl Thanks! This is cool and I'm looking forward to it. But it's of course still a bit rough around the edges. Should bugs like this be reported?
![image](https://user-images.githubusercontent.com/25161793/125340484-ceb93600-e352-11eb-9071-1429483a6a4c.png)

**Comment by joyceerhl** [comment]
Yes--could you please file a new issue with the sample code that produced that plot? Since we use the same output renderers for notebooks and interactive window, this is likely a problem in notebooks as well.
